src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -147,6 +148,51 @@ def rename_fs_volume(self, volname, newvolname, sure):
 
         return rename_volume(self.mgr, volname, newvolname)
 
+    def volume_info(self, **kwargs):
+        ret     = None
+        volname = kwargs['vol_name']
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                path = self.volspec.base_dir
+                vol_info_dict = {}
+                try:                
+                    st = fs_handle.statx(path.encode('utf-8'), cephfs.CEPH_STATX_SIZE,
+                            cephfs.AT_SYMLINK_NOFOLLOW)
+                            
+                    usedbytes = st['size']
+                    vol_info_dict = info(path)
+                    vol_info_dict['size'] = f"{int(usedbytes)} B"
+                except cephfs.Error as e:
+                    if e.args[0] == errno.ENOENT:
+                        pass
+                df = self.mgr.get("df")
+                pool_stats = dict([(p['id'], p['stats']) for p in df['pools']])
+                osdmap = self.mgr.get("osd_map")
+                pools = dict([(p['pool'], p) for p in osdmap['pools']])
+                metadata_pool_id, data_pool_ids = get_pool_ids(self.mgr, volname)
+                vol_info_dict["pools"] = {"metadata" : [], "data" : []}
+                for pool_id in [metadata_pool_id] + data_pool_ids:
+                    if pool_id == metadata_pool_id:
+                        pool_type = "metadata"
+                    else:
+                        pool_type = "data"
+                    vol_info_dict["pools"][pool_type].append({
+                                    'name' : pools[pool_id]['pool_name'],
+                                    'used': f"{pool_stats[pool_id]['bytes_used'] / (10**9)} GiB",
```

Comment:
I think this is `GB` not `GiB` ? https://massive.io/file-transfer/gb-vs-gib-whats-the-difference/

Reply:
While testing the PR, I got this output, the metadata usage shown in exponential form?. Please check it out below.

```
{
    "mon_addrs": [
        "192.168.1.12:40826"
    ],
    "pools": {
        "data": [
            {
                "avail": "106.288914432 GiB",
                "name": "cephfs.a.data",
                "used": "0.0 GiB"
            }
        ],
        "metadata": [
            {
                "avail": "106.288914432 GiB",
                "name": "cephfs.a.meta",
                "used": "3.2768e-05 GiB"    <<< THIS IS NOT READER FRIENDLY
            }
        ]
    }
}

```

Reply:
Handle divide by zero exception ?

Reply:
Used formatting upto precision of `9` as all the value are being shown with 9 digits after decimal.